hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -423,6 +423,19 @@ public static Path getTableDir(Path rootdir, final TableName tableName) {
         tableName.getQualifierAsString());
   }
 
+  /**
+   * Returns the {@link org.apache.hadoop.fs.Path} object representing the region directory under
+   * path rootdir
+   *
+   * @param rootdir qualified path of HBase root directory
+   * @param tableName name of table
+   * @param regionName The encoded region name
+   * @return {@link org.apache.hadoop.fs.Path} for region
+   */
+  public static Path getRegionDir(Path rootdir, TableName tableName, String regionName) {
```

Comment:
Should hbase-common know about region dirs? Seems like it knows about table dirs so I suppose this ok.